### PR TITLE
Create Whatsapp setup document

### PR DESCRIPTION
**Pull request by jpwalden**
Signed-off-by: jari-pekka walden <jari-pekka.walden@jolla.com>

### CODE REVIEW

File: Support/Help_Articles/Whatsapp_Setup/README.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+parent: Help Articles
+layout: default
+nav_order: 10000
+---
+
+Please note that the Sailfish OS must be in the very latest version in order for all features described in this article to work. Read about updating your software [**here**](/Support/Help_Articles/Updating_Sailfish_OS/). These instructions were updated for Sailfish OS version 4.1.0 and Whatsapp version 2.21.7.14 but they will be precise enough for some earlier and later versions, too.
+
+WhatsApp application (like most Android apps) get updates on a regular basis. We recommend installing these updates as after some time the app and Sailfish OS might become incompatible.
+
+NOTE: The implementation of Android App Support varies by device. On the older devices, i.e., Jolla Phone, Jolla Tablet, Jolla C and Xperia X, Android App Support simulates Android version 4, whereas, in the case of the more recent products, Xperia XA2 and Xperia 10, the simulated Android version is 10.  Also, the related Sailfish menu pages for Android Support have some differences between those two support categories. Bear this in mind when reading [this chapter](#Making sure WhatsApp gives notifications even when not open).
```
[the remark this answers is not in the document]
Comment (jpwalden) [reply Support/Help_Articles/Whatsapp_Setup/README.md]: Thanks for the comments, now this should be fixed with hyphens.

File: Support/Help_Articles/Whatsapp_Setup/README.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+* <a href="WA_003b.png"><img src="WA_003b.png" alt="Setting up the WA"></a>
+  <span class="md_figcaption">
+  </span>
+* <a href="WA_006b.png"><img src="WA_006b.png" alt="Setting up the WA"></a>
+  <span class="md_figcaption">
+  </span>
+  Screen captures from setting up the whatsapp 
+</div>
+
```
[the remark this answers is not in the document]
Comment (jpwalden) [reply Support/Help_Articles/Whatsapp_Setup/README.md:69]: Good catch, don't know who's phone number that is, but good to remove that

File: Support/Help_Articles/Whatsapp_Setup/README.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+parent: Help Articles
+layout: default
+nav_order: 10000
+---
+
+Please note that the Sailfish OS must be in the very latest version in order for all features described in this article to work. Read about updating your software [**here**](/Support/Help_Articles/Updating_Sailfish_OS/). These instructions were updated for Sailfish OS version 4.1.0 and Whatsapp version 2.21.7.14 but they will be precise enough for some earlier and later versions, too.
+
+WhatsApp application (like most Android apps) get updates on a regular basis. We recommend installing these updates as after some time the app and Sailfish OS might become incompatible.
+
+NOTE: The implementation of Android App Support varies by device. On the older devices, i.e., Jolla Phone, Jolla Tablet, Jolla C and Xperia X, Android App Support simulates Android version 4, whereas, in the case of the more recent products, Xperia XA2 and Xperia 10, the simulated Android version is 10.  Also, the related Sailfish menu pages for Android Support have some differences between those two support categories. Bear this in mind when reading [this chapter](#Making-sure-WhatsApp-gives-notifications-even-when-not-open).
```
Comment (vigejolla) [inline Support/Help_Articles/Whatsapp_Setup/README.md:13]: Still doesn't work. You missed that part in @jovirkku 's comment about lower case only.

File: Support/Help_Articles/Whatsapp_Setup/README.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+* <a href="Screenshot-17-03-27-16-33-15.png" class="narrow-image"><img src="Screenshot-17-03-27-16-33-15.png" alt="Setting-up-wa"></a>
+  <span class="md_figcaption">
+  </span>
+</div>
+
+2) In the next view, press the three dots top-right and select Help:
+
+<div class="flex-images" markdown="1">
+
+* <a href="Screenshot-17-03-27-16-36-20.png" class="narrow-image"><img src="Screenshot-17-03-27-16-36-20.png" alt="Setting-up-wa"></a>
```
Comment (vigejolla) [inline Support/Help_Articles/Whatsapp_Setup/README.md:241]: This image doesn't exist, maybe you forgot to `git add` it?

Comment (jpwalden) [reply Support/Help_Articles/Whatsapp_Setup/README.md:241]: Yes, that happened (for some reason forgot to download to the folder, my bad, now added.

File: Support/Help_Articles/Whatsapp_Setup/README.md (hunk 3 above)
Comment (jpwalden) [reply Support/Help_Articles/Whatsapp_Setup/README.md]: yes, missed that part, hopefully now..

File: Support/Help_Articles/Whatsapp_Setup/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jpwalden) [reply Support/Help_Articles/Whatsapp_Setup/README.md:69]: > Picture "WA_006b" has your(?) phone number fully visible.
> 
> At this point, after entering the code from the SMS, the two-step verification dialogue will appear (if 2-step v. has been enabled for this WhatsApp account). It requires typing the 6-digit code. Should we mention this or not?

Good addition, added that.

File: Support/Help_Articles/Whatsapp_Setup/README.md [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+parent: Help Articles
+layout: default
+nav_order: 10000
+---
+
+Please note that the Sailfish OS must be in the very latest version in order for all features described in this article to work. Read about updating your software [**here**](/Support/Help_Articles/Updating_Sailfish_OS/). These instructions were updated for Sailfish OS version 4.1.0 and Whatsapp version 2.21.7.14 but they will be precise enough for some earlier and later versions, too.
+
+WhatsApp application (like most Android apps) get updates on a regular basis. We recommend installing these updates as after some time the app and Sailfish OS might become incompatible.
+
+NOTE: The implementation of Android App Support varies by device. On the older devices, i.e., Jolla Phone, Jolla Tablet, Jolla C and Xperia X, Android App Support simulates Android version 4, whereas, in the case of the more recent products, Xperia XA2 and Xperia 10, the simulated Android version is 10.  Also, the related Sailfish menu pages for Android Support have some differences between those two support categories. Bear this in mind when reading [this chapter](#making-sure-whatsApp-gives-notifications-even-when-not-open).
```
Comment (vigejolla) [inline Support/Help_Articles/Whatsapp_Setup/README.md:13]: There's still one capital A there

File: Support/Help_Articles/Whatsapp_Setup/README.md [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+
+# Downloading WhatsApp
+If you have not used WhatsApp before you are reading the correct instructions here. Carry on.
+
+However, if you have had WhatsApp on some phone previously and you would like to migrate it to a new phone then [jump here](#moving-your-whatsApp-data-to-a-new-phone).
```
Comment (vigejolla) [inline Support/Help_Articles/Whatsapp_Setup/README.md:18]: capital A here as well

File: Support/Help_Articles/Whatsapp_Setup/README.md [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+
+## On the new phone
+1. Install WhatsApp application.
+2. Open "Settings > Apps > WhatsApp"
+3. If you have opened the Whatsapp app on this phone already then tap the "Clear data" button first. This will pave way for the data that we will copy to this phone soon.
+4. Tap "Open Android settings"
+5. Tap Permissions
+6. Tap Storage to allow Whatsapp to read and write your previous (and future) data on the phone.
+7. Tap all other items you need for using WhatsApp. For full service, you will need to give all permissions.  If you do not give the permissions now the app will keep prompting for them later. See the 
+three pictures at step 5 of [this chapter](#making-sure-whatsApp-gives-notifications-even-when-not-open) above.
```
Comment (vigejolla) [inline Support/Help_Articles/Whatsapp_Setup/README.md:157]: and here

File: Support/Help_Articles/Whatsapp_Setup/README.md [hunk 8]
```diff
@@ -0,0 +1,252 @@
...
+
+(*) We are looking for alternative ways to do this easily.
+
+# Troubleshooting
+
+**Q: I can't get all my contacts to show in WhatsApp. Some show names, but some just show the phone number**
+
+On Sailfish 4 and on the latest releases of Sailfish 3, the contacts saved in the People app should automatically appear in WhatsApp, given that you have given the permission for this in the settings of WhatsApp.
+
+A1: If you have an Xperia XA2 or Xperia 10, then follow step #5 in [this chapter](#making-sure-whatsApp-gives-notifications-even-when-not-open) to give the required permissions. Should this not help, carry on reading the instructions just below the next two pictures.
```
Comment (vigejolla) [inline Support/Help_Articles/Whatsapp_Setup/README.md:200]: and here

File: Support/Help_Articles/Whatsapp_Setup/README.md [hunk 9]
```diff
@@ -0,0 +1,252 @@
...
+  </span>
+</div>
+
+3) Please try out the steps suggested in the next view.
+
+If your problems continue even after these steps, we can only suggest you check in the People app that contact information for these certain contacts are in order (e.g. no multiple numbers, incorrect numbers).
+
+**Q: I can't send pictures or videos. Whatsapp reports "this picture does not exist in the phone memory"**
+
+Did you forget to run the commands at the end of [this chapter](#moving-your-whatsApp-data-to-a-new-phone)? That should help.
```
Comment (vigejolla) [inline Support/Help_Articles/Whatsapp_Setup/README.md:252]: and here

Comment (jpwalden) [reply Support/Help_Articles/Whatsapp_Setup/README.md:252]: damn, I need to get bigger screen.. how I missed those..

Comment (jpwalden) [reply Support/Help_Articles/Whatsapp_Setup/README.md]: Now those finally should be ok..

File: Support/Help_Articles/Whatsapp_Setup/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jovirkku) [reply Support/Help_Articles/Whatsapp_Setup/README.md:69]: Thanks

[the remark this answers is not in the document]
Comment (jovirkku) [reply Support/Help_Articles/Whatsapp_Setup/README.md:69]: LGTM

**Review by jovirkku — APPROVED** [review]
LGTM (I already did this but...)

File: Support/Help_Articles/Whatsapp_Setup/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jovirkku) [reply Support/Help_Articles/Whatsapp_Setup/README.md:69]: ok

**Review by vigejolla — APPROVED** [review]

**Pull request merged**